client/src/lib/utils/faucetRequest.ts

```diff
@@ -1,57 +1,54 @@
 import { PUBLIC_DEMO_MODE as DEMO } from "$env/static/public";
+
 import type { NetworkData } from "./networkData";
 
 export async function request(
-	address: string,
-	recaptcha: string,
-	network: NetworkData,
-	parachain?: number
+  address: string,
+  recaptcha: string,
+  network: NetworkData,
+  parachain?: number,
 ): Promise<string> {
-	if (DEMO) {
-		return boilerplateRequest(address, recaptcha);
-	}
-	const chain = parachain && parachain > 0 ? parachain.toString() : undefined;
-	return faucetRequest(address, recaptcha, network, chain);
+  if (DEMO !== undefined && DEMO !== "") {
+    return await boilerplateRequest(address);
+  }
+  const chain = parachain && parachain > 0 ? parachain.toString() : undefined;
+  return await faucetRequest(address, recaptcha, network, chain);
```

Comment:
Do we get any benefit by returning the await of a promise vs returning the promise itself?

Reply:
We get +1 line in stacktraces, which might shed light on the error origin in some cases.  
I wouldn't call myself a big fan of the rule, but I would like to keep it rather than to change it.